s3/src/main/java/com/github/awsjavakit/s3/S3Driver.java

```diff
@@ -147,6 +147,11 @@ public String readFile(URI uri) {
     return getFile(filePath);
   }
 
+  public InputStream readFileAsStream(URI uri) {
```

Comment:
Do you really want a URI, or rather an "S3 bucket path"? The latter being a specialisation of the former